pkg/base1/cockpit.css

```diff
@@ -314,15 +314,27 @@ a {
 }
 
 /* Our terminal or logs */
+.console-ct-container {
+    border: none;
+}
+
 .console-ct {
     font-family: Menlo, Monaco, Consolas, monospace;
-    margin-top: 0;
-    margin-bottom: 0;
+    background-color: black;
+    margin: 0;
     font-size: 10px;
-    text-align: center;
     line-height: 1;
 }
 
+.console-ct-fullscreen {
+    position: absolute;
+    top: 44px;
```

Comment:
It's relative to the page. The 44px is the header, which is also hard-coded to be 44px high.

This is the best way I've found to determine the height of the page. I'm very open to changing that if there's a better way.